<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin_reportes</class>
 <widget class="QWidget" name="admin_reportes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lb_total">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>700</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Roboto&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>1021</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(236, 238, 238);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="sideBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>768</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(244, 247, 244);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>90</y>
     <width>701</width>
     <height>35</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btn_ventas">
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">*{
	border: none;
	color: #A6A9A5;
	margin-left: 10px;
	margin-right: 10px;
}

:hover{
	color: #585858;
}

</string>
      </property>
      <property name="text">
       <string>Ventas</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_transacciones">
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">*{
	border: none;
	color: #A6A9A5;
	margin-left: 10px;
	margin-right: 10px;
}

:hover{
	color: #585858;
}
</string>
      </property>
      <property name="text">
       <string>Transacciones almacen</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_comandas">
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">*{
	border: none;
	color: #A6A9A5;
	margin-left: 10px;
	margin-right: 10px;
}

:hover{
	color: #585858;
}

</string>
      </property>
      <property name="text">
       <string>Comandas</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_buscarReporte">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>687</y>
     <width>207</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	color: white;
	border-radius: 10px;
	background-color: #46B04A;
	border: none;
	padding: 5px;
}

:hover{
	background-color: white;
	border: 1px solid #46B04A;
	color: #46B04A;
}</string>
   </property>
   <property name="text">
    <string>Buscar reporte</string>
   </property>
  </widget>
  <widget class="QLabel" name="tipoUsuario">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>132</y>
     <width>145</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(133, 133, 133);
</string>
   </property>
   <property name="text">
    <string>Inicio fecha apertura</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>298</x>
     <y>29</y>
     <width>721</width>
     <height>35</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="Encabezados2">
    <item>
     <widget class="QLabel" name="tipoUsuario_4">
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">*{
	border: none;
	color: #A6A9A5;
	margin-left: 10px;
	margin-right: 10px;
}</string>
      </property>
      <property name="text">
       <string>[NOMBRE_ADMIN]</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="tipoUsuario_6">
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">*{
	border: none;
	color: #A6A9A5;
	margin-left: 10px;
	margin-right: 10px;
}</string>
      </property>
      <property name="text">
       <string>Reportes</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="elPatio_4">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>28</y>
     <width>195</width>
     <height>56</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #585858
</string>
   </property>
   <property name="text">
    <string>Buscar reportes bajo demanda</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="openExternalLinks">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>130</y>
     <width>721</width>
     <height>631</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="comandas">
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="topMargin">
      <number>9</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>9</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0" colspan="2">
      <widget class="QScrollArea" name="scrollArea">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="cursor" stdset="0">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContents</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollareaalbum">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>710</width>
          <height>611</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="leftMargin">
          <number>15</number>
         </property>
         <property name="topMargin">
          <number>15</number>
         </property>
         <property name="rightMargin">
          <number>15</number>
         </property>
         <property name="bottomMargin">
          <number>15</number>
         </property>
         <property name="spacing">
          <number>20</number>
         </property>
         <item row="1" column="0" colspan="4">
          <widget class="QLabel" name="line_2">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>1</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(236, 238, 238);
</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QWidget" name="verticalWidget" native="true">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>150</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>150</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(244, 247, 244);
border-radius: 20px;</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>10</number>
            </property>
            <property name="leftMargin">
             <number>10</number>
            </property>
            <property name="topMargin">
             <number>10</number>
            </property>
            <property name="rightMargin">
             <number>10</number>
            </property>
            <property name="bottomMargin">
             <number>10</number>
            </property>
            <item>
             <widget class="QLabel" name="tipoUsuario_3">
              <property name="font">
               <font>
                <family>Roboto</family>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(133, 133, 133);
</string>
              </property>
              <property name="text">
               <string>Número de Comandas</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lb_totalComandas_2">
              <property name="font">
               <font>
                <family>Roboto</family>
                <pointsize>40</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(133, 133, 133);
margin-top: 20px;
margin-bottom: 20px;
margin-left: 10px;
</string>
              </property>
              <property name="text">
               <string>[TOTAL_COMANDAS]</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QWidget" name="verticalWidget_2" native="true">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>150</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>150</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(244, 247, 244);
border-radius: 20px;</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="tipoUsuario_5">
              <property name="font">
               <font>
                <family>Roboto</family>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(133, 133, 133);
</string>
              </property>
              <property name="text">
               <string>Número de platillos</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lb_totalPlatillos">
              <property name="font">
               <font>
                <family>Roboto</family>
                <pointsize>40</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(133, 133, 133);
margin-top: 20px;
margin-bottom: 20px;
margin-left: 10px;
</string>
              </property>
              <property name="text">
               <string>[TOTAL_PLATILLOS]</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QWidget" name="verticalWidget_3" native="true">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>150</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>150</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(244, 247, 244);
border-radius: 20px;</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="tipoUsuario_8">
              <property name="font">
               <font>
                <family>Roboto</family>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(133, 133, 133);
</string>
              </property>
              <property name="text">
               <string>Número de Personas</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lb_totalPeronas">
              <property name="font">
               <font>
                <family>Roboto</family>
                <pointsize>40</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(133, 133, 133);
margin-top: 20px;
margin-bottom: 20px;
margin-left: 10px;
</string>
              </property>
              <property name="text">
               <string>[TOTAL_COMANDAS]</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0" colspan="4">
          <layout class="QGridLayout" name="gridLayout_3">
           <property name="sizeConstraint">
            <enum>QLayout::SetMaximumSize</enum>
           </property>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="transaccionesAlmacen"/>
   <widget class="QWidget" name="ventas"/>
  </widget>
  <widget class="QDateEdit" name="de_incio">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>163</y>
     <width>175</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="displayFormat">
    <string>dd / MMM / yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateEdit" name="de_fin">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>237</y>
     <width>175</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="displayFormat">
    <string>dd / MMM / yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>191</width>
     <height>125</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>Archivo de reporte</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <property name="spacing">
     <number>10</number>
    </property>
    <item>
     <widget class="QLabel" name="tipoUsuario_9">
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(133, 133, 133);
</string>
      </property>
      <property name="text">
       <string>Formato</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="cbox_pdf">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>PDF</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbox_excel">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>EXCEL</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="btn_buscarReporte_2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">*{
	color: white;
	border-radius: 10px;
	background-color: #46B04A;
	border: none;
	padding: 5px;
}

:hover{
	background-color: white;
	border: 1px solid #46B04A;
	color: #46B04A;
}</string>
      </property>
      <property name="text">
       <string>Generar Reporte</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="tipoUsuario_7">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>206</y>
     <width>130</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(133, 133, 133);
</string>
   </property>
   <property name="text">
    <string>Fin fecha apertura</string>
   </property>
  </widget>
  <zorder>sideBar</zorder>
  <zorder>lb_total</zorder>
  <zorder>btn_buscarReporte</zorder>
  <zorder>tipoUsuario</zorder>
  <zorder>widget_3</zorder>
  <zorder>elPatio_4</zorder>
  <zorder>stackedWidget</zorder>
  <zorder>widget</zorder>
  <zorder>line</zorder>
  <zorder>dateEdit</zorder>
  <zorder>dateEdit</zorder>
  <zorder>tipoUsuario</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget_2</zorder>
  <zorder>de_incio</zorder>
  <zorder>de_fin</zorder>
  <zorder>tipoUsuario_7</zorder>
  <zorder>groupBox</zorder>
  <zorder>cbox_pdf</zorder>
  <zorder>cbox_excel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
